dalite/views/errors.py

```diff
@@ -3,12 +3,13 @@
 
 import logging
 
+from django.http import HttpResponse
 from django.template.response import TemplateResponse
 
 logger = logging.getLogger("dalite")
 
 
-def response_400(req, msg="", logger_msg="", log=None):
+def response_400(req, msg="", logger_msg="", log=None, use_template=True):
```

Comment:
Out of curiosity, when would we not want a TemplateResponse?

Reply:
These functions are there to make sure error responses are always logged. If they're implemented through the middleware, there's no way to modify the log message.

Reply:
When it's an error message returned for an ajax call, we don't want it to be a template.

Reply:
Ah, ok.